geode-core/src/test/java/org/apache/geode/test/junit/rules/GfshCommandRule.java

```diff
@@ -265,4 +267,8 @@ public File getWorkingDir() {
   public enum PortType {
     locator, jmxManager, http
   }
+
+  public CommandResult getCommandResult() {
+    return commandResult;
```

Comment:
instead of expose this result, it would be better to add another method in CommandResultAssert object to assert what you need to do.